adaptors/library/jobs/commcare-asri-submitxls.js

```diff
@@ -0,0 +1,8 @@
+// Update the kunjungan case with a new row of data. The row contains the case_id and satusehat_id
+
+submitXls([{ case_id: 'abcde', satusehat_id: '12345' }], {
```

Comment:
Since this is just a one-liner, would it be better if it were just a regular example in the adaptor docs?

Reply:
Yes, its already in the adaptor docs Let me remove it